packages/malloy/src/lang/ast/types/malloy-element.ts

```diff
@@ -264,6 +272,13 @@ export abstract class MalloyElement {
     this.log(`INTERNAL ERROR IN TRANSLATION: ${msg}`);
     return new Error(msg);
   }
+
```

Comment:
Creating the list bugs me for some reason. I'd be happier if either

1) Preferred. The walk gesture was re-written so a return value from any node short circuited the walk which then make needs a simple call to walk.
2) Talk me into. The tree-to-flatlist happens is only part of the implementation of needs